hkm/migrations/0041_merge_20201124_0947.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.8 on 2020-11-24 07:47
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('hkm', '0040_auto_20200903_0708'),
+        ('hkm', '0040_remove_obsolete_image_fields_from_record'),
```

Comment:
Is this the way Django wanted to handle the merge conflict? I would have guessed that the migration file name prefixes would have (or are preferred) to be unique, so we wouldn't have `0040` twice.

Reply:
Yup, I actually renamed the file to `0041_update_showcase_albums_description.py`, I'm just thinking that does this make the `merge` migration obsolete 🤔

Reply:
Probably yes 😅

Reply:
Can you try if it works without the merge stuff?

Reply:
Yeah, I'll try to remove it.

Reply:
Seems to be working. Removed the merge file and edited `0041_update_showcase_albums_description` file's dependency to `0040_remove_obsolete_image_fields_from_record`